README.md

```diff
@@ -159,9 +159,44 @@ Of course, this can both be combined with `first`, `last`, `before`, and `after`
 
 **Important:**
 If your app regularly orders by another column, you might want to add a database index for this.
-Say that your order column is `author` then your index should be on `CONCAT(author, '-', id)`.
+Say that your order column is `author` then you'll want to add a compound index on `(author, id)`.
+If your table is called `posts` you can use a query like this in MySQL or Postgres:
+```sql
+CREATE INDEX index_posts_on_author_and_id ON posts (author, id);
+```
+Or you can just do it via an `ActiveRecord::Migration`:
+```ruby
+class AddAuthorAndIdIndexToPosts < ActiveRecord::Migration
+  def change
+    add_index :posts, %i[author id]
+  end
+end
+```
 
 Please take a look at the _"How does it work?"_ to find out more why this is necessary.
+          
+#### Order by more complex logic
+
+Sometimes you might not only want to oder by a column ascending or descending, but need more complex logic.
+Imagine you would also store the author's role on the `posts` table (for simplicity's sake).
+And the role could be `member`, `owner`, or `editor`.
+Then you might want to show all posts by the `owner` first, followed by the ones by `editor`s and lastly followed by the ones by regular `member`s.
```

Comment:
The author's role example is a little bit difficult to understand, because the role is not on the post, but maybe on the application.
We could use as an example post categories (intro, general, networking, ...) or status (pending, approved, rejected)

Reply:
Good point – I'll change the example for categories!